hail/python/test/hailtop/batch/test_batch_service_backend.py

```diff
@@ -507,6 +507,21 @@ def test_specify_job_region(service_backend: ServiceBackend):
     assert res_status['state'] == 'success', str((res_status, res.debug_info()))
 
 
+def test_use_default_region_when_not_specifying_any_regions(service_backend: ServiceBackend):
+    b = batch(service_backend)
+    j = b.new_job('region')
+    default_region = service_backend.default_region()
```

Comment:
It would probably be good to assert this is "us-central1" somehow just to make absolutely sure it has the behavior we want either in the test or double checking the global config secret.

Reply:
I don't think we want to hardcode config in the tests, for the sake of people running forks. But I did just confirm that the default is "us-central1" in the global config secret.